### PR TITLE
Animate preview show/hide 

### PR DESCRIPTION
**Pull request by asdil12**
As requested in #719 comments

### CODE REVIEW

**Comment by coolo** [comment]
does this work for you @aaannz ?

File: assets/javascripts/result_preview.js [hunk 1]
```diff
@@ -29,12 +29,12 @@ function preview(a, force) {
...
             }
             $('#preview_container_out').css('display', 'block').css('height', $('#preview_container_in').height());
-            $('body, html').stop(true, true).animate({scrollTop: a.offset().top-3, queue: false}, 0);
+            $('body, html').stop(true, true).animate({scrollTop: a.offset().top-3, queue: false}, 250);
         });
     }
     else {
         // hide
-        $('#dummy_space').hide();
+        $('#dummy_space').hide(300);
```
Comment (okurz) [inline assets/javascripts/result_preview.js]: magic numbers…

Comment (coolo) [inline assets/javascripts/result_preview.js]: the css is full of magic colours too :)

Comment (okurz) [inline assets/javascripts/result_preview.js]: http://clean-code-developer.de/die-grade/roter-grad/#Die_Pfadfinderregel_beachten

**Pull request merged**